### PR TITLE
Add FIND Command

### PR DESCRIPTION
**Pull request by thewrik**
Support for tags added

### CODE REVIEW

**Comment by ian-from-dover** [comment]
To show that you are a seasoned github pro, you should name future PRs using active tense. So the name should fit in the sentence:
When applied, this PR will `PR name` eg. `Update find command`

Just fix these 2 style errors then should be able to merge.
```
Error: eckstyle] [ERROR] /home/runner/work/tp/tp/src/main/java/seedu/address/model/person/PersonContainsKeywordsPredicate.java:22:25: '(' should be on the previous line. [SeparatorWrap]
[26](https://github.com/AY2122S2-CS2103T-T11-4/tp/runs/5505736044?check_suite_focus=true#step:8:26)

[27](https://github.com/AY2122S2-CS2103T-T11-4/tp/runs/5505736044?check_suite_focus=true#step:8:27)
Error: eckstyle] [ERROR] /home/runner/work/tp/tp/src/main/java/seedu/address/model/person/PersonContainsKeywordsPredicate.java:22:113: '||' should be on a new line. [OperatorWrap]
```

**Review by ian-from-dover — APPROVED** [review]
LGTM. Good use of streams 👍

File: src/test/java/seedu/address/model/person/PersonContainsKeywordsPredicateTest.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+        // Multiple keywords with tags, with non-matching tag
+        predicate = new PersonContainsKeywordsPredicate(Arrays.asList("Alice", "Bob", "intern", "tech"));
+        assertFalse(predicate.test(new PersonBuilder().withName("Alice Bob").withTags("Intern").build()));
+
+        // Keywords match phone, email and address, but does not match name or tags
+        predicate = new PersonContainsKeywordsPredicate(
+                Arrays.asList("12345", "alice@email.com", "Main", "Street"));
+        assertFalse(predicate.test(new PersonBuilder().withName("Alice").withPhone("12345")
+                .withEmail("alice@email.com").withAddress("Main Street").build()));
```
Comment (ian-from-dover) [inline src/test/java/seedu/address/model/person/PersonContainsKeywordsPredicateTest.java:87]: I'm a little confused here. Should the predicate contain tags? Should the new person built contain different tags? Should the keyword predicate have a name at all?

If all is as it should be then feel free to merge.

**Review by ian-from-dover — COMMENTED** [review]
Just see the comment about the last test, if it is as it should be, i'll approve tonight.

File: src/test/java/seedu/address/model/person/PersonContainsKeywordsPredicateTest.java (hunk 1 above)
Comment (thewrik) [reply src/test/java/seedu/address/model/person/PersonContainsKeywordsPredicateTest.java:87]: The Predicate here contains a bunch of non-name non-tag words, all of which are in other fields of the given person. Hope that helps.

**Pull request merged**